doc/whats_new.rst

```diff
@@ -65,6 +65,7 @@ API
 - :func:`mne_bids.make_dataset_description` now takes the argument `overwrite` which will reset all fields if `True`. If `False`, user-provided fields will no longer be overwritten by :func:`mne_bids.write_raw_bids` when its `overwrite` argument is `True` unless new values are supplied, by `Alex Rockhill`_ (`#478 <https://github.com/mne-tools/mne-bids/pull/478>`_)
 - :func:`mne_bids.make_report` is now available from the `mne_bids` namespace that creates a string output of a summary of the BIDS dataset. In addition, the command line interface allows one to call `make_report`, by `Adam Li`_ (`#457 <https://github.com/mne-tools/mne-bids/pull/457>`_)
 - Added namespace :code:`mne_bids.path` which hosts path-like functionality for MNE-BIDS by `Adam Li`_ (`#483 <https://github.com/mne-tools/mne-bids/pull/483>`_)
+- :func:`mne_bids.path._parse_bids_filename` is now part of the public API and has consequently been renamed to `:func:`mne_bids.path._parse_bids_filename`, by `Richard Höchenberger`_ (`#487 <https://github.com/mne-tools/mne-bids/pull/487>`_)
```

Comment:
private functions cannot be put in the what's new as they are not part of the public API

Reply:
you also need to add to API rst page the new public function.

Reply:
> private functions cannot be put in the what's new as they are not part of the public API

But it is now… so it's a "new" function. Do you mean I should not list it under "API", but instead under Changelog?



> you also need to add to API rst page the new public function.

will do